src/test/resources/pages/instructorCourseDetailsWithSections.html

```diff
@@ -160,168 +160,7 @@ <h1>
-          Tut Group 2
-        </td>
-        <td id="studentteam-c0.1.1">
-          Team 2
-        </td>
-        <td id="studentname-c0.2">
-          Benny Charles
-        </td>
-        <td class="align-center">
-          Yet to join
-        </td>
-        <td id="studentemail-c0.2">
-          benny.tmms@gmail.tmt
-        </td>
-        <td class="no-print align-center">
-          <a class="btn btn-default btn-xs margin-bottom-7px" data-original-title="View the details of the student" data-placement="top" data-toggle="tooltip" href="/page/instructorCourseStudentDetailsPage?courseid=CCDetailsUiT.CS2103&studentemail=benny.tmms%40gmail.tmt&user=CCDetailsUiT.instr2&token=${sessionToken}" rel="noopener noreferrer" target="_blank" title="">
-            View
-          </a>
-          <a class="btn btn-default btn-xs margin-bottom-7px" data-original-title="Use this to edit the details of this student. <br>To edit multiple students in one go, you can use the enroll page: <br>Simply enroll students using the updated data and existing data will be updated accordingly" data-placement="top" data-toggle="tooltip" href="/page/instructorCourseStudentDetailsEdit?courseid=CCDetailsUiT.CS2103&studentemail=benny.tmms%40gmail.tmt&user=CCDetailsUiT.instr2&token=${sessionToken}" rel="noopener noreferrer" target="_blank" title="">
-            Edit
-          </a>
-          <a class="course-student-remind-link btn btn-default btn-xs margin-bottom-7px" data-original-title="Email an invitation to the student requesting him/her to join the course using his/her Google Account. Note: Students can use TEAMMATES without ‘joining’, but a joined student can access extra features e.g. set up a user profile" data-placement="top" data-toggle="tooltip" href="/page/instructorCourseRemind?pagenameremindstudentisfrom=instructorCourseDetailsPage&courseid=CCDetailsUiT.CS2103&studentemail=benny.tmms%40gmail.tmt&user=CCDetailsUiT.instr2&token=${sessionToken}" title="">
-            Send Invite
-          </a>
-          <a class="course-student-delete-link btn btn-default btn-xs margin-bottom-7px" data-course-id="CCDetailsUiT.CS2103" data-original-title="Delete the student and the corresponding submissions from the course" data-placement="top" data-student-name="Benny Charles" data-toggle="tooltip" href="/page/instructorCourseStudentDelete?courseid=CCDetailsUiT.CS2103&studentemail=benny.tmms%40gmail.tmt&user=CCDetailsUiT.instr2&token=${sessionToken}" title="">
-            Delete
-          </a>
-          <a class="btn btn-default btn-xs margin-bottom-7px" data-original-title="View all data about this student" data-placement="top" data-toggle="tooltip" href="/page/instructorStudentRecordsPage?courseid=CCDetailsUiT.CS2103&studentemail=benny.tmms%40gmail.tmt&user=CCDetailsUiT.instr2&token=${sessionToken}" rel="noopener noreferrer" target="_blank" title="">
-            All Records
-          </a>
-        </td>
-      </tr>
-    </tbody>
-  </table>
```

Comment:
@ritisha2000 You missed out this comment :)

> You shouldn't need to update src/test/resources/pages/* files. Those are legacy files that we are trying to deprecate.

Reply:
Sorry about that. I have taken out the changes I made. Thank you for reminding me.